abe/abe.go

```diff
@@ -12,38 +12,38 @@ import (
 	"github.com/fentec-project/gofe/sample"
 )
 
-// abeParams represents configuration parameters for the ABE scheme instance.
-type abeParams struct {
+// ABEParams represents configuration parameters for the ABE scheme instance.
+type ABEParams struct {
```

Comment:
You can leave it as `abeParams` here since it doesn't need to be exported. I only meant to change `Abe` to `ABE` specifically so we don't change the functionality.

Reply:
Ok, done.